proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
+* Tends to not commit to longer term backwards compatibility 
+
+== CN4J
+Pros
+
+* Allows for parties interested in both MPWG and CN4J/Jakarta to sign one working group participation agreement
+* Addresses IP gap in existing MP development
+* Tends to focus on stability and backwards compatibility 
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+
+Cons
+
+* Currently has a higher entry cost and overall overhead to run
+* Currently has no zero entry participation model for larger companies to sponsor specification development
+* Introduces a conflict of interest between evolution and support of enterprise customers and their previous investment in Java EE with a shared steering committee
+* Has more process around separate steering, specification and marketing committees
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+* Has baggage to overcome due to legal requirements to move Java EE to Jakarta 
```

Comment:
Don't understand this point and how it applies to MP within a new CNJWG

Reply:
I think this is referring to the javax namespace issue, and the documentation source for the remaining Specifications coming from Oracle.  Once we get all of that in place, then the legal requirements should be behind us.

Reply:
The javax to jakarta namespace transformation is one that has to be done, and it adds no value to the end user. It is not clear end users will be willing to move beyond Jakarta EE 8 because of this. This poses a risk for MP joining a shared working group.

Reply:
Agreed and it's another reason why I suggested even a delayed merging of MP and Jakarta EE if the community decided to go down the 1 WG route.